dpnp/dpnp_iface_indexing.py

```diff
@@ -1395,9 +1344,16 @@ def put_along_axis(a, ind, values, axis):
     values : {scalar, array_like}, (Ni..., J, Nk...)
         Values to insert at those indices. Its shape and dimension are
         broadcast to match that of `ind`.
-    axis : int
+    axis : {None, int}
         The axis to take 1d slices along. If axis is ``None``, the destination
         array is treated as if a flattened 1d view had been created of it.
+    mode : {"wrap", "clip"}, optional
+        Specifies how out-of-bounds indices will be handled. Possible values
+        are:
+        - ``"wrap"``: clamps indices to (``-n <= i < n``), then wraps
```

Comment:
For the documentation to be shown correctly.
```suggestion
        are:
        
        - ``"wrap"``: clamps indices to (``-n <= i < n``), then wraps
```